pages/coinbase/signin.py

```diff
@@ -0,0 +1,28 @@
+from selenium import webdriver
+from selenium.webdriver.support import expected_conditions as EC  
+from selenium.webdriver.support.ui import WebDriverWait as wait
+from selenium.webdriver.common.by import By
+import secure.cookies
+import secure.creds
+import config
+import time
+
+
+class Sign_In_Page():
+
+    def __init__(self,driver, get_SignIn_url: bool=None):  #if the driver is starting at the sign in page use keyword argument True at instantiation 
```

Comment:
Ooh. Maybe the cookie injection should go in init. You could put an optional argument in here and if you put in a cookie file it adds those cookies during init. Consider this; it's probably better than putting it in the sign in button, but you should try it first and choose the best solution.